src/commands.rs

```diff
@@ -771,7 +772,17 @@ pub fn run_command(cmd: Command) -> Result<i32> {
             trace!("Command::Compile {{ {:?}, {:?}, {:?} }}", exe, cmdline, cwd);
             let jobserver = unsafe { Client::new() };
             let conn = connect_or_start_server(get_port(), startup_timeout)?;
-            let mut runtime = Runtime::new()?;
+
+            let worker_threads = env::var("SCCACHE_CLIENT_WORKER_THREADS")
```

Comment:
if this is a new variable, please update the doc

Reply:
Done 👍